docs/pages/guides/_meta.json

```diff
@@ -12,12 +12,6 @@
     "title": "Create a private collection",
     "display": "hidden"
   },
-  "create-nft": {
-    "title": "Create an NFT",
-    "display": "hidden"
-  },
-  "sell-nft": {
-    "title": "Sell an NFT",
-    "display": "hidden"
-  }
+  "create-nft": "Creating an NFT",
+  "sell-nft": "Selling an NFT"
```

Comment:
The pages are not displayed in the menu on the preview: https://docs-git-docs-create-sell-nft-liteflow.vercel.app/guides/setup-fees
Did I miss anything?

Reply:
Sometime you need to force rebuild the doc, there is some issue with the cache on vercel. Just launched a new build